tests/CMakeLists.txt

```diff
@@ -37,11 +37,10 @@ if (NOT NO_STATIC_ANALYSIS)
 endif ()
 
 # C options
-set(CMAKE_C_STANDARD 11)
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Werror -pedantic -fstrict-aliasing")
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wdouble-promotion -Wswitch-enum -Wfloat-equal -Wundef")
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wconversion -Wtype-limits")
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wsign-conversion -Wcast-align -Wmissing-declarations")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wsign-conversion -Wcast-align -Wmissing-declarations -Wno-unused-local-typedefs")
```

Comment:
Whenever you disable one warning, a kitten gets sad :smirk_cat: 

There are two PETA-approved solutions to this:

- The good one: move that one `_Static_assert` in canard_dsdl.c out of the function scope so that its typedef is no longer function-local.
- The bad one: move your `-Wno-unused-local-typedefs` to the test matrix such that it is only disabled when using C99.